### PR TITLE
My quizzes

### PR DESCRIPTION
**Pull request by elacymerys**
Create My Quizzes page template for teacher's view

### CODE REVIEW

File: thesis-gui/src/components/my-quizzes/MyQuizzes.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                <h2>{ name }</h2>
+                <p>{ `${questionsNumber} question(s)` }</p>
+            </IonLabel>
+        </IonItem>
+    );
+}
+
+const QuizzesList: React.FC = () => {
+    const quizzesListItems = QUIZZES.map(quiz => {
+        return <QuizzesListItem name={ quiz.name } questionsNumber={ quiz.questionsNumber } />
```
Comment (MaksymilianK) [inline thesis-gui/src/components/my-quizzes/MyQuizzes.tsx]: Can be simplified by removing `return`.

File: thesis-gui/src/components/my-quizzes/MyQuizzes.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    return (
+      <IonList lines="full" style={{ paddingTop: "15px" }}>
+          { quizzesListItems }
+      </IonList>
+  );
+}
+
+export const MyQuizzes: React.FC = () => {
+    return (
```
Comment (MaksymilianK) [inline thesis-gui/src/components/my-quizzes/MyQuizzes.tsx]: same as above

File: thesis-gui/src/components/my-quizzes/QuizCreator.tsx [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        <IonItem>
+            <IonInput placeholder={ `Enter answer ${answerNumber}` }></IonInput>
+            <IonRadio slot="end"></IonRadio>
+        </IonItem>
+    );
+}
+
+const AnswersInput: React.FC = () => {
+    const answerInputItems = Array.from({ length: 4 }, (_, i) => i + 1).map(answerNumber => {
+        return <AnswerInputItem answerNumber={ answerNumber } />
```
Comment (MaksymilianK) [inline thesis-gui/src/components/my-quizzes/QuizCreator.tsx]: Can be simplified by removing `return`.

File: thesis-gui/src/components/my-quizzes/QuizCreator.tsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+    return (
+        <IonItem>
+            <IonInput placeholder={ `Enter answer ${answerNumber}` }></IonInput>
+            <IonRadio slot="end"></IonRadio>
+        </IonItem>
+    );
+}
+
+const AnswersInput: React.FC = () => {
+    const answerInputItems = Array.from({ length: 4 }, (_, i) => i + 1).map(answerNumber => {
```
Comment (MaksymilianK) [inline thesis-gui/src/components/my-quizzes/QuizCreator.tsx]: I didn't know that we can generate an Array of numbers in JS this way, so I learned something new :) . But to be honest, for such a small array `[0,1,2,3].map(...` looks more readable :) .

**Review by MaksymilianK — CHANGES_REQUESTED** [review]

File: thesis-gui/src/components/my-quizzes/MyQuizzes.tsx (hunk 2 above)
Comment (elacymerys) [reply thesis-gui/src/components/my-quizzes/MyQuizzes.tsx]: ?

Comment (MaksymilianK) [reply thesis-gui/src/components/my-quizzes/MyQuizzes.tsx]: `return `can be removed

**Review by MaksymilianK — DISMISSED** [review]

**Review by MaksymilianK — APPROVED** [review]

**Pull request merged**